root/static/styles/favicons.less

```diff
@@ -34,6 +34,7 @@
 .favicon("genius");
 .favicon("googleplus", 32);
 .favicon("ibdb");
+.favicon("IMDb");
```

Comment:
This shouldn't have the `, 32` like googleplus and instagram etc.? (I don't know what that value is, I just noted that the imdb.png is called -32 while the vmdb one is -16…)